blocks/library/gallery/block.js

```diff
@@ -149,13 +150,14 @@ class GalleryBlock extends Component {
 							<MediaUploadButton
 								buttonProps={ {
 									className: 'components-icon-button components-toolbar__control',
-									'aria-label': __( 'Edit Gallery' ),
+									'aria-label': editButtonLabel,
```

Comment:
Would be nice if we could infer this from the `tooltip` passed to `MediaUploadButton`, but I guess the upload button is not always an icon button, so not so simple.

Reply:
Actually I was planing to revisit the `MediaUploadButton` to use render Props instead, that way we'd just use an `IconButton` in the render prop. 

The thinking is that the `MediaUpload` should only deal with `MediaUpload` and not the UI to trigger the modal.